relay-metrics/src/aggregatorservice.rs

```diff
@@ -73,14 +40,17 @@ pub enum Aggregator {
     AcceptsMetrics(AcceptsMetrics, Sender<bool>),
     /// Merge the buckets.
     MergeBuckets(MergeBuckets),
-
-    /// Message is used only for tests to get the current number of buckets in `AggregatorService`.
+    /// Message is used only for tests to get the current number of buckets in the default [`AggregatorService`].
     #[cfg(test)]
     BucketCountInquiry(BucketCountInquiry, Sender<usize>),
 }
 
 impl Interface for Aggregator {}
 
+/// Check whether the aggregator has not (yet) exceeded its total limits. Used for health checks.
+#[derive(Debug)]
+pub struct AcceptsMetrics;
+
```

Comment:
Could you move the definition of this struct and the one below after all implementations of `Aggregator`, to keep them together?